config_example.sh

```diff
@@ -3,3 +3,15 @@
 # Get a valid pull secret (json string) from
 # You can get this secret from https://cloud.openshift.com/clusters/install#pull-secret
 export PULL_SECRET=''
+
+# Additional DNS
+export ADDN_DNS='1.1.1.1'
```

Comment:
Can we default all of these in common.sh, at the top before the per-user configs, so these are optional?

Then we can leave these commented and say they're optional overrides?

Reply:
Also should we default ADDN_DNS to an empty string, then we use whatever we get via DHCP by default?